lib/routes/token-codes.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const errors = require('../error')
+const isA = require('joi')
+const validators = require('./validators')
+const HEX_STRING = validators.HEX_STRING
+
+module.exports = (log, db, customs) => {
+  return [
+    {
+      method: 'POST',
+      path: '/tokenCodes/verify',
+      config: {
+        validate: {
+          payload: {
```

Comment:
The tests suggest this should take a `metricsContext` payload param as well, but it doesn't appear to.